docs/robot_advanced.rst

```diff
@@ -117,32 +117,68 @@ With the ``PageObjects`` library, you can define classes that represent page obj
-* ``self.salesforce``: A reference to the Salesforce keyword library. Call any keyword in this library by converting the name to all lowercase, and replacing all spaces with underscores (such as ``self.salesforce.wait_until_loading_is_complete``).
-* ``self.selenium``: A reference to SeleniumLibrary. Call any keyword in this library by converting the name to all lowercase, and replacing all spaces with underscores (such as ``self.selenim.wait_until_page_contains_element``).
+The following example illustrates how to create a ``Listing`` page object for ``CustomObject__c``.
+
+.. code-block:: python
+
+    from cumulusci.robotframework.pageobjects import ListingPage, pageobject
 
+    @pageobject(page_type='Listing', object_name='CustomObject__c')
+    class CustomObjectListingPage(ListingPage):
+        ...
+
+Using object aliases
+""""""""""""""""""""
+
+Within a test, if you want to refer to the page object with a more human-readable name such as ``Custom Object`` rather than ``CustomObject__c`` you can do so by setting ``object_name`` to ``Custom Object`` and then defining ``_object_name`` in the class, like in the following example.
+
+.. code-block:: python
+
+    from cumulusci.robotframework.pageobjects import ListingPage, pageobject
+
+    @pageobject(page_type = 'Listing', object_name = 'My Object')
+    class CustomObjectListingPage(ListingPage):
+        _object_name = 'MyObject__c'
+        ...
+
+By using an alias, you can reference the page object with either the alias or the actual object name. For example, the following two uses of ``Go to page`` are identical:
+
+.. code-block:: robot
+
+    Go to page  Listing  My Object
+    Go to page  Listing  MyObject__c
 
 
 Page Object Base Classes
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-Presently, CumulusCI provides the following base classes, which should be used for all classes that use the ``pageobject`` decorator:
+Presently, CumulusCI provides the following base classes, which should be used for all classes that use the ``pageobject`` decorator. These base classes may be imported from ``cumulusci.robotframework.pageobjects``.
 
-* `cumulusci.robotframework.pageobjects.BasePage <Keywords.html#file-cumulusci/robotframework/pageobjects/BasePageObjects.py>`_ A generic base class used by the other base classes. Use the ``BasePage`` class for creating custom page objects when none of the other base classes make sense.
-    * The ``BasePage`` adds the ``Log current page object`` keyword to every page object. This keyword is most useful when debugging tests. It will add to the log information about the currently loaded page object.
+* `cumulusci.robotframework.pageobjects.BasePage <Keywords.html#file-cumulusci/robotframework/pageobjects/BasePageObjects.py>`_ A generic base class used by the other pageobject classes. Use the ``BasePage`` class for creating custom page objects when none of the other base classes make sense.
```

Comment:
What are the circumstances that determine when none/some of the other base classes make sense?

Reply:
Thanks for pointing this out. I've tried rewording it to make it slightly more clear. My assumption is that if someone is experienced enough to be creating custom page objects, they'll understand the concept of base classes.